blog/estymacja/index.md

```diff
@@ -0,0 +1,165 @@
+
+## Koszulki w praniu
+
+Po warsztacie, pełni zapału ruszyliśmy estymować. Dodatkowo uzgodniliśmy, że:
+* Nie będziemy rejestrować godzin faktycznie spędzonych na danym zadaniu - wystarczy określić jaki to typ zadania dokumentacyjnego i przyporządkować mu rozmiar. 
+    Uznaliśmy, że rejestrowanie godzin zbytnio nas obciąży, szczególnie gdy nasza praca wymaga częstego przełączania się między zadaniami. 
+* Jeśli w trakcie pracy okaże się, że przeszacowaliśmy lub niedoszacowaliśmy, poprawialiśmy oryginalną estymatę tak, żeby była bliższa prady.
+* Będziemy na rezultaty patrzeć kwartalnie.
+
+Po pierwszym kwartale wyszło nam, że umiemy już zmierzyć ile czasu poświęciliśmy na zadania dokumentacyjne łącznie.
+
+Po drugim kwartale zaobserwowaliśmy nie tylko spory wzrost zadań ale też i ich rozmiarów. 
+
+Trzeci kwartał powiedział nam z więcej na temat tego ile średnio zadań i jakiej wielkości  jesteśmy w stanie zmieścić w kwartale. 
+
+### Rekalibracja
+
+Ponadto, ustaliliśmy z zespołem, że potrzebna będzie pewna rekalibracja naszego oryginalnego systemu. 
+Na przykład doszliśmy do wniosku, że mamy za duży przeskok pomiędzy zadaniem S (5 godzin) a M (15 godzin). 
+Planujemy kolejny warsztat 😅
+
+## Wnioski
+
+Ten eksperyment nadal trwa. Już jednak stał się stałym elementem procesu i daje nam jako zespołowi informacje, które po dłuższym czasie będzie można wykorzystać w rozmowach z różnymi *stakeholderami*.
+
+Znamy rozmiar zadań ukończonych i zadań w przygotowaniu oraz ich wartość godzinową. 
+
+Znamy realne obłożenie pracą - w wymiarze godzinowym. To daje nam możliwość dalszej analizy czy to się wszystko jeszcze spina - po tym jak policzymy liczbę godzin dostępnych na pracę (odejmując urlopy, spotkania zespołowe, 1:1, etc.). 
+
+Nadal jednak, zaczynając nowy kwartał nie mamy pojęcia jak ostatecznie się on rozwinie. Budowa produktu przebiega wielotorowo i często zależymy od precyzji planowania i estymacji zespołów produktowych i deweloperskich. 
+
+Co innego w krótszej perspektywie, kiedy wiemy:
+* Ile mamy dostępnego czasu Technical Writera? 
+* Ile jakiego typu zadań jest do udokumentowania? 
+* Do kiedy?
+
+Wtedy jak najbardziej możemy określić czy jest ryzyko, czy jednak damy radę.
+
+Estymacja może być niezwykle przydatnym procesem, jeśli wiemy po co chcemy ją wdrożyć. W naszym przypadku, pierwotna potrzeba czyli zdolność określenia czy zobowiązujemy się do dostarczenia dokumentacji okazała się niemożliwa w perspektywie kwartału, ale zadziałała w kontekście krótszych ram czasowych i bliższych deadlinów. 
+
+Dodatkowo mamy mocne, obiektywne, bo oparte na danych argumenty pomocne w przypadku rozmów dotyczących zatrudnienia nowych Writerów. 
+
+Pewnym nieplanowanym elementem było również to, że zespół miał okazję wspólnie skatalogować i przedyskutować każdy typ zadania dokumentacyjnego, wypowiedzieć się na temat czasu jaki im on zajmuje, porównać swoje argumenty z innymi i wymienić się *best practices* na optymalizację, pewnych etapów 🙌
```

Comment:
Cy zebraliście tez jakoś statystykę z tego o ile się pomyliliście w szacowaniu, tzn. o ile się zmieniły estymaty vs. actuals? Nie wiem czy warto to opisywać, ale jestem ciekaw.

Reply:
Po to między innymi planujemy rekalibrację, żeby mieć czas zrobić mały zespołowy krok wstecz i przemyśleć ile razy w ciągu kwartału zdarzyło się nam wracać do estymacji i ją aktualizować. Do tej pory niestety nie daliśmy rady ogarnąć takiego spotkania :/
Zdarza się jednak, że rozmawiamy o estymacji na 1:1, kiedy Writer wyestymuje wg mnie za mało lub za dużo, wtedy robimy sync i dogadujemy stanowiska :)